docs/basics/basic_knowledge.md

```diff
@@ -0,0 +1,16 @@
+# Basic Knowledge
```

Comment:
It probably doesn't make sense to keep such short page around - integrate it elsewhere.